RT_SYCL/build_parameters.hpp

```diff
@@ -0,0 +1,12 @@
+#ifndef BUILD_PARAMETERS_HPP
+#define BUILD_PARAMETERS_HPP
+namespace buildparams {
+#if USE_SINGLE_TASK
+constexpr bool SINGLE_TASK = true;
```

Comment:
Since it is not a macro
```suggestion
constexpr bool use_single_task = true;
```
?

Reply:
done in a49de312